README.md

```diff
@@ -1,23 +1,40 @@
 # Kids First-Sentieon Joint Cohort Calling
 
 This is a work in progress and under active development.
-An efficient, fast, and cost effective workflow for joint calling cohorts up to ~4000 individuals on the CAVATICA platform
+An efficient, fast, and cost effective workflow for joint calling cohorts up to ~4000 individuals on the CAVATICA platform.
+You will likely need to have installed [sbpack](https://pypi.org/project/sbpack/) to push the necessary apps into your project, and it is recommended to have the python packages listed in `python_pip_requirements.txt` installed.
+Both workflows have limited implementations of the [`GVCFtyper` algo](https://support.sentieon.com/manual/usages/general/#gvcftyper-algorithm) which is part of the [Sentieon driver library](https://support.sentieon.com/manual/usages/general/#driver-binary).
+For any question/comments/support request, please email support@kidsfirstdrc.org
+<p align="center">
+ <img src="https://github.com/d3b-center/d3b-research-workflows/raw/master/doc/kfdrc-logo-sm.png" alt="data service logo"/>
+</p>
 
-### Badges
+<p align="center">
+<a href="https://github.com/kids-first/Kids-First-Sentieon-Joint-Cohort-Genotyping-Workflow/blob/main/LICENSE"><img src="https://img.shields.io/github/license/kids-first/kf-template-repo.svg?style=for-the-badge"></a>
+</p>
 
-Update the LICENSE badge to point to the new repo location on GitHub.
-Note that the LICENSE badge will fail to render correctly unless the repo has
-been set to **public**.
+## [Preferred Cohort Calling Workflow](docs/KF_COHORT_JG_SINGLE_WF.md)
+This workflow runs on the CAVATICA platform with few parameters necessary to run.
+The platform has an internal workflow scatter limit of ~2200 files, which means, **if your cohort size is greater than this, it will not run!**
+To load this app into your project, follow the instructions in the [sbpack](https://pypi.org/project/sbpack/) install to load `workflow/kf-joint-cohort-call-by-chr-wf.cwl`
 
-Add additional badges for CI, docs, and other integrations as needed within the
-`<p>` tag next to the LICENSE.
+## [Large Cohort Calling Workflow](docs/KF_NOTEBOOK_JG_WORKFLOW.md)
+This notebook can be run locally or copied to a CAVATICA Data Studio Analysis session.
+If your cohort is >2200 samples, but projected to use less than 4TB per split job and chromosome call job, you can use this.
+To estimate the amount of disk space required, we recommend taking the average file size of your cohort, and assuming even distribution of file size by chromosome, multiply it by the proportion if the size of the genome to be used, and double it. For example, if the average file size were 7.2 GB, and, using chr 1-22,X,Y as our total genome, chr1 is the largest and take up about 0.08 of total bases, and you want to leave room for the joint call result for that chromosome so:
```

Comment:
```suggestion
To estimate the amount of disk space required, we recommend taking the average file size of your cohort, and assuming even distribution of file size by chromosome, multiply it by the proportion of the size of the genome to be used, and double it. For example, if the average file size is 7.2 GB, and considering chromosomes 1-22, X, and Y as the entire genome with chromosome 1 being the largest and accounting for approximately 8% of the total bases, you should allocate enough space for the joint call result for that chromosome:
```

Just a rephrase because the original is kind of difficult to read